src/main/java/org/opensrp/web/rest/SettingResource.java

```diff
@@ -114,7 +114,7 @@ public void setSettingService(SettingService settingService) {
 
 			for (int i = 0; i < clientSettings.length(); i++) {
 
-				dbSettingsArray.put(settingService.saveSetting(clientSettings.getString(i).toString()));
+				dbSettingsArray.put(settingService.saveSetting(clientSettings.get(i).toString()));
```

Comment:
Since the method expects a string and ..toString is being called why not simplify it to 
```
dbSettingsArray.put(settingService.saveSetting(clientSettings.getString(1)));
```

Reply:
This method receives a json payload instead of a string in this upload. Getting a Json as getString fails.